Misc/NEWS.d/next/Library/2017-07-24-23-52-39.bpo-30931.mZMsJ7.rst

```diff
@@ -0,0 +1,3 @@
+asyncore: poll() and poll2() functions now check if a dispatcher was closed (if
+it's file descriptor is no more in the map) before calling its handlers:
+handlers of closed dispatchers are no more called.
```

Comment:
poll() and poll2() already check if a dispatcher was closed before calling it. The change in behavior is detecting a replaced dispatcher using same fd, and skipping it instead of calling it before it is ready.